node/lib/util/repo_status.js

```diff
@@ -258,12 +261,35 @@ class Submodule {
             return false;
         }
 
-        // Otherwise, check its index status.
+        // Finally, we're clean if there is no change to our index status,
+        // which would indicate a URL change.
 
         return null === this.d_indexStatus;
     }
 
     /**
+     * Return true if this submodule has been added.
+     * @return {Boolean}
+     */
+    isNew() {
+        return FILESTATUS.ADDED === this.d_indexStatus;
+    }
+
+    /**
+     * Return true if this submodule cannot be committed, having no commits or
```

Comment:
It's generally better to have functions defined in positive terms (isCommittable rather than isUncommittable).